script/Ciel/Day 7/07_01_CIEL07_8.txt

```diff
@@ -17,6 +17,7 @@ I finally let my legs stumble to a halt,
 [sha:04fd958e9d9cb1d19b7955c684026f85915db121]{
 -- Page 623, Offset 16306.
 -- 一方、あのお姫様はと言うと、
+// that makes it three
```

Comment:
I guess we can revert it if we want, but I'm not a stickler for consistency

Reply:
My vote is to revert but I'm fine keeping the first one

Reply:
On second thought, it's fine to leave as is at least for this entire segment